examples/3d/perspective_zoom.rs

```diff
@@ -0,0 +1,125 @@
+//! Shows how to zoom and orbit a perspective projection camera.
+
+use std::ops::Range;
+
+use bevy::{input::mouse::AccumulatedMouseScroll, prelude::*};
+
+#[derive(Debug, Default, Reflect, Resource)]
+struct CameraSettings {
+    // Multiply keyboard inputs by this factor
+    pub orbit_speed: f32,
+    // Clamp fixed vertical scale to this range
+    pub zoom_range: Range<f32>,
+    // Multiply mouse wheel movements by this factor
+    pub zoom_speed: f32,
+}
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .init_resource::<CameraSettings>()
+        .add_systems(Startup, (setup, instructions))
+        .add_systems(Update, camera_controls)
+        .register_type::<CameraSettings>()
+        .run();
+}
+
+/// Set up a simple 3D scene
+fn setup(
+    mut camera_settings: ResMut<CameraSettings>,
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    camera_settings.orbit_speed = 0.02;
+    // Clamp FOV to this range. Note that we can't adjust FOV to more than PI,
```

Comment:
```suggestion
    // Clamp FOV to this range. Note that we can't adjust FOV to more than π,
```
Love me some unicode ;)